f/missionConditions/fn_SetWeather.sqf

```diff
@@ -30,7 +30,6 @@ _MissionLightnings = 0;
 _MissionWindStr = 0.01;
 _MissionWindGusts = 0.01;
 _MissionWaves = 0;
-_MissionHumidity = 0;
```

Comment:
This variable `_MissionHumidity` has been here forever, but it was never used.
Probably because the `simulSetHumidity` command has been removed, so there's no way to set the humidity.